dev/VSIX/ProjectTemplates/Desktop/CSharp/SingleProjectPackagedApp/MainWindow.xaml

```diff
@@ -9,7 +9,11 @@
     mc:Ignorable="d"
     Title="$projectname$">
 
-    <StackPanel Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Center">
-        <Button x:Name="myButton" Click="myButton_Click">Click Me</Button>
-    </StackPanel>
+    <Window.SystemBackdrop>
```

Comment:
@haonanttt Since we are enabling `Mica`, we need to make sure that we remove the following from all of the `Blank Page` item templates:

```xml
Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
```
A page should not have a solid background brush for `Mica` to pass through.

Reply:
Searched in the repo, only 1 template have this setting, `Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"` is removed from Blank Page item template.